### DISCUSSION TITLE
Camera won't start today - error 403

### DISCUSSION THREAD

**Discussion by simondsmason** [QuantumEntangledAndy/neolink#331] [Q&A] [answered]
Working fine for a long time.  Just noticed that the stream was down and checked the docker.  It won't start - gives a 403 error.  Seeing this in the log:

```
[2024-11-24T15:47:04Z WARN  neolink::common::pushnoti] Issue connecting to push notifications server: ServerError("PHONE_REGISTRATION_ERROR")
[2024-11-24T15:47:09Z WARN  neolink::common::pushnoti] Issue connecting to push notifications server: Http(reqwest::Error { kind: Decode, source: Error("expected value", line: 1, column: 1) })
[2024-11-24T15:47:12Z WARN  neolink::common::pushnoti] Issue connecting to push notifications server: ServerError("PHONE_REGISTRATION_ERROR")
```

I have a really simple setup with one Reolink E1:

```
# A bind value of 0.0.0.0 means any network this computer can access
# You can change this to a specfic network e.g. "192.168.1.101" here
# Or to no networks e.g. this computer only "127.0.0.1"
bind = "0.0.0.0"

# Default port is 8554 but you can change it by uncommenting the following
# bind_port = 8554

[[cameras]]
name = "garage_inside"
username = "admin"
password = "********"
address = "192.168.2.32:9000"
```

Not sure what I did to mess this up?  Thanks for any help.

**Comment by simondsmason** [comment]
bump - is this supported anymore?  Thanks.

**Reply by simondsmason** [reply]
I am still getting these error messages but the camera is now working?  How do I get rid of this error?  Thanks

**Comment by QuantumEntangledAndy** [comment, marked as answer]
The push notification is unsupported because of changes google made upstream. You should disable it in the config `push_notifications = false` (if i recall correctly). It is being removed in my dev branches atm, trying to find some replacement. I believe that the offical NVR can get a signal to trigger a record from a battery camera just need someone with an NVR to help me bounce ideas off

**Reply by simondsmason** [reply]
Thanks.
